### ISSUE TITLE
Buf breaking action fails when Git LFS is enabled

### ISSUE THREAD

**Issue by charithe** [bufbuild/buf#424]
To provide some context, this is a continuation of an issue I previously raised in the `buf-breaking-action` repository about LFS: https://github.com/bufbuild/buf-breaking-action/issues/10. That issue was fixed very quickly. However, it looks like the recent change to use bare clones (https://github.com/bufbuild/buf/commit/503f6318162f2fb83b9063aa69073e3000ccd7cb) has broken LFS again. When the breaking action runs with buf 0.51.0, it fails with the following error:

```
Error downloading object: my_file_in_lfs.png (c8889f0): Smudge error: Error downloading my_file_in_lfs.png (c8889f0a9786553413d49edbbab93739de2988e8b9f8f33df8d1a7368453e715): batch request: missing protocol: "")
```

I suspect this is because there's no remote available for the LFS filter to download the contents from.

**Comment by bufdev** [comment]
Sorry for this. We're working on it.

**Comment by bufdev** [comment]
This is fixed on v0.51.1.
